### PR TITLE
fix package security issues

### PR DESCRIPTION
**Pull request by leozc**
### What changed? Why?
Fix package security issues detected by security bot
 
### How did you test the change?
Build and run

### CODE REVIEW

**Comment by wangwzhou** [comment]
Wondering did you run `make codegen` to generate those proto go files? why the protoc version changed in those pb.go files

File: protos/coinbase/c3/common/common.pb.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.25.0
-// 	protoc        v3.13.0
+// 	protoc        v3.21.12
```
Comment (wangwzhou) [inline protos/coinbase/c3/common/common.pb.go:4]: Wondering did you run `make codegen` to generate those proto go files? why the protoc version changed in those pb.go files

Comment (leozc) [reply protos/coinbase/c3/common/common.pb.go:4]: it seems it is related to the mac m1 protoc version
https://github.com/coinbase/chainstorage/pull/6/files#diff-ad8a239f2d9110a9929810a189865ad0515e31355df3088588fccbb94879ce4f
see `brew install protobuf@3.21.9`

Comment (wangwzhou) [reply protos/coinbase/c3/common/common.pb.go:4]: Should we pin the protoc version to 3.21.9 before checking this change? I mean should we firstly merge the previous PR you referred here?

Comment (wangwzhou) [reply protos/coinbase/c3/common/common.pb.go:4]: @jiezhang any concern to check in the previous PR to bump protoc?

Comment (wangwzhou) [reply protos/coinbase/c3/common/common.pb.go:4]: @leozc How about combining your previous protoc version change in this PR as well?

**Pull request closed without merging**